execution_engine/src/shared/transform.rs

```diff
@@ -360,9 +365,12 @@ impl From<&Transform> for casper_types::Transform {
             Transform::Write(StoredValue::Bid(bid)) => {
                 casper_types::Transform::WriteBid(bid.clone())
             }
-            Transform::Write(StoredValue::Withdraw(unbonding_purses)) => {
+            Transform::Write(StoredValue::Unbonding(unbonding_purses)) => {
                 casper_types::Transform::WriteWithdraw(unbonding_purses.clone())
             }
+            Transform::Write(StoredValue::Withdraw(_)) => casper_types::Transform::Failure(
+                "withdraw purses should not be be written to global state".to_string(),
```

Comment:
`StoredValue::Withdraw` is the only variant that fails this transformation. Can you explain why it should not be written to global state?

Reply:
`WithdrawPurses` are the old representation of UnbondingPurses, therefore moving forward we shouldn't write those variants to global state